### ISSUE TITLE
testrules.bat error, LanguageTool-20160408-snapshot

### ISSUE THREAD

**Issue by MikeUnwalla** [languagetool-org/languagetool#388]
LanguageTool-20160408-snapshot introduced a problem with testrules:
```
D:\LanguageTool-3.4-SNAPSHOT>testrules en
Error: Could not find or load main class org.languagetool.rules.patterns.PatternRuleTest
Error: Could not find or load main class org.languagetool.tagging.disambiguation.rules.DisambiguationRuleTest
Error: Could not find or load main class org.languagetool.rules.bitext.BitextPatternRuleTest
Error: Could not find or load main class org.languagetool.ValidateFalseFriendsXmlTest
```
The error is in LanguageTool-20160413-snapshot.zip. I did not check other snapshots between the 2 dates.

**Comment by danielnaber** [comment]
This is caused by https://github.com/languagetool-org/languagetool/pull/385 which makes `junit.jar`and `hamcrest-core.jar` disappear from the `libs` directory when running `mvn package`. @aldenquimby any idea? I didn't have time yet to really understand the implications the dependency convergence enforcement.

**Comment by danielnaber** [comment]
This is fixed now (by reverting #385 for now) so the fix should be in tonight's snapshot.
